my-pern-app/backend/src/server.ts

```diff
@@ -44,10 +44,10 @@ app.put('/api/updateMovie/:movieId', async (req, res, next)=>{
         const {newTitle, newSummary, newImdbLink, newRating} = req.body
         const sql = `
         update "movies"
-        set "title" = $2
-        set "summary" = $3
-        set "imdbLink" = $5
-        set "rating" = $5
+        set "title" = $2,
+            "summary" = $3,
+            "imdbLink" = $4,
+            "rating" = $5
```

Comment:
Line 37:
You wrote this endpoint as if you are using error handling middleware, but you do not have any error handling middleware configured. If you don't install it (as the last `app.use()` call) then you will need to return a status 500 response on your own.